src/java.base/share/classes/sun/net/www/MimeTable.java

```diff
@@ -381,9 +382,7 @@ public Properties getAsProperties() {
     }
 
     protected boolean saveAsProperties(File file) {
-        FileOutputStream os = null;
-        try {
-            os = new FileOutputStream(file);
+        try (FileOutputStream os = new FileOutputStream(file)) {
```

Comment:
This is not strictly equivalent as now an exception thrown during `close()` will be printed instead of being ignored. I have no idea whether it's a good thing or a bad thing, but that's a _different_ thing...
So unless someone with more historical knowledge of this area can comment, maybe this change should be reverted.

Reply:
I believe it's always good to have at least _some_ trace if we got Exception. If closing FileOutputStream failed - there is no guarantee, that all written content is actually stored to filesystem.
BTW, this method `saveAsProperties` seems unused...

Reply:
This is legacy code with probably poor testing coverage. Usually we don't print anything on the console directly and unconditionally - except in some well identified cases. Ignoring exceptions thrown by `close` is a common idiom. That said - it does seem that this method is indeed never called - so for the sake of simplification I'd agree to keep this change as proposed.
Remind me to sponsor this after the fork.

Reply:
>Remind me to sponsor this after the fork.
fork is done :)

Reply:
/sponsor

Reply:
Looks like `/sponsor` in comment is not considered as a command by bot